docs/resources/dsm_csr.md

```diff
@@ -1,16 +1,15 @@
-# dsm\_certs
+# dsm\_csrs
 
-## dsm\_certs
+## dsm\_csrs
 
 Returns the Fortanix DSM cert security object certs from the cluster as a Resource.
```

Comment:
Should we update this comment to say DSM CSR(certificate signing request) instead of certs